src/edit/main.js

```diff
@@ -512,6 +512,15 @@ export class ProseMirror {
   markAllDirty() {
     this.dirtyNodes.set(this.doc, DIRTY_REDRAW)
   }
+
+  // :: (string) → string
+  // Return a translated string, if a translate function has been supplied,
+  // or the original string.
+  translate(string) {
+    return (typeof this._translate === "function")
+      ? this._translate(string)
+      : `${string}`
```

Comment:
Silly me, it's left from testing. Will fix.

On 2016-03-07 16:32, Marijn Haverbeke wrote:
>> @@ -512,6 +512,15 @@ export class ProseMirror {
>>    markAllDirty() {
>>      this.dirtyNodes.set(this.doc, DIRTY_REDRAW)
>>    }
>> +
>> +  // :: (string) → string
>> +  // Return a translated string, if a translate function has been supplied,
>> +  // or the original string.
>> +  translate(string) {
>> +    return (typeof this._translate === "function")
>> +      ? this._translate(string)
>> +      : `${string}`
> 
> Why not just `string`? I.e. why the wrapping template string?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ProseMirror/prosemirror/pull/248/files#r55219224
> 

-- 
http://yesbabyyes.se/
+46 709 89 03 85